src/solvers/dg.jl

```diff
@@ -212,6 +244,33 @@ function Base.show(io::IO, ::MIME"text/plain", integral::SurfaceIntegralStrongFo
 end
 
 
+"""
+    SurfaceIntegralUpwind(splitting)
+
+Couple elements with upwind simulataneous approximation terms (SATs)
+that use a particular `FluxSplitting`.
```

Comment:
```suggestion
Couple elements with upwind simultaneous approximation terms (SATs)
that use a particular flux `splitting`.
```

Reply:
Already included in my local working copy of https://github.com/trixi-framework/Trixi.jl/pull/1273